CONTRIBUTING.md

```diff
@@ -89,10 +91,13 @@ We use:
 - TypeDoc - for documentation
 
 ### Bots
+
 - Dependabot Preview - makes PRs for dependency upgrades
 - Codecov - creates code coverage report for each PR, works with Circle CI
 
 ### Github Actions
+

```

Comment:
the random md spaces reminded me: try running eclint over the repo before merging this in: https://github.com/VandyHacks/vaken/issues/75#issuecomment-486396561